packages/pointer_interceptor/pointer_interceptor_ios/example/pubspec.yaml

```diff
@@ -0,0 +1,88 @@
+name: pointer_interceptor_ios_example
+description: "Demonstrates how to use the pointer_interceptor_ios plugin."
+# The following line prevents the package from being accidentally published to
```

Comment:
The boilerplate comments should all (except the one on the `path` dependency) be scrubbed out.